ios-drag-drop.js

```diff
@@ -136,8 +136,10 @@
       var touch = event.changedTouches[0];
       var x = touch[coordinateSystemForElementFromPoint + 'X'];
       var y = touch[coordinateSystemForElementFromPoint + 'Y'];
-      dropEvt.offsetX = x - target.x;
-      dropEvt.offsetY = y - target.y;
+      dropEvt.clientX = x;
+	  dropEvt.clientY = y;
+	  dropEvt.offsetX = x - event.target.clientLeft;
+	  dropEvt.offsetY = y - event.target.clientTop;
```

Comment:
It probably is not, what really matters its the clientx and clienty, as they are part of the drop event spec from what I read. I believe offset could be removed in favor of screen x and y.

https://developer.mozilla.org/en-US/docs/Web/Events/drop